plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayer.java

```diff
@@ -164,14 +165,14 @@ private void addSymbolLayer(String layerId, String beforeLayerId) {
     layer.setProperties(
       iconAllowOverlap(true),
       iconIgnorePlacement(true),
-      iconSize(zoom(
-        exponential(
-          stop(22f, iconSize(1f)),
-          stop(12f, iconSize(1f)),
-          stop(10f, iconSize(0.6f)),
-          stop(0f, iconSize(0.6f))
-        ).withBase(1f)
-      )),
+      iconSize(interpolate(
+        exponential(1f), zoom(),
+        stop(22f, iconSize(1f)),
```

Comment:
You shouldn’t nest the `iconSize` property function in there.

cc @cammace @tobrun